### ISSUE TITLE
Remove click as dependency

### ISSUE THREAD

**Issue by ericfrederich** [bneijt/poetry-lock-docker#1]
I thought I'd give this a try to bundle up an application I wrote using the latest click version.
But... since this package itself is using click, we get this problem.

Given the nature of what this tool is trying to do, it would probably be best not to take on any dependency at all and use `argparse` or something from standard library.  Otherwise you'll continue to run into this problem.

```
$ poetry add poetry-lock-package
Using version ^0.3.1 for poetry-lock-package

Updating dependencies
Resolving dependencies... (0.0s)

  SolverProblemError

  Because poetry-lock-package (0.3.1) depends on click (>=7.1.2,<8.0.0)
```

**Comment by bneijt** [comment]
Fixed in release 0.4.0, but I still have to change the current project to reflect the new `--no-parent` to `--no-root` change between 0.3 and 0.4. At that point I will  close the issue in this project as well.

Thank you for reporting and I hope it now works as intended!

**Comment by ericfrederich** [comment]
Thanks, just realized I reported this on the wrong project ;-)
